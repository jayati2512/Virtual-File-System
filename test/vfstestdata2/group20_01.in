createvfs demo 1000
mountvfs demo
makedir / a
makedir /a/b/c/d/e/f/g/ h
makedir /p/q/r/s/t/u/v/ w
movedir /a/b/c/d/ /a/b/c/d/e/
listdir / 1 /tmp/outputdir 
makedir /x/y/z aa/a
makedir /aa/bb/cc/dd/ee/ff kk
makedir /aa/bb/ cc
deletedir /p
listdir /p 0 /tmp/outputdir
listdir / -5 /tmp/outputdir
unmountvfs demo
mountvfs demo
makedir /p/q/r/s/t/u/v/ w
movedir /p /a/b/c
movedir 
unmountvfs demo
mountvfs demo
unmountvfs demo
mountvfs demo
makedir /home/a/b/f/d e
makedir /home/a/b/d/d f
makedir /home/a/b/g/d f
deletedir /home/a/b
listdir /home 1 /tmp/outputdir
unmountvfs demo
