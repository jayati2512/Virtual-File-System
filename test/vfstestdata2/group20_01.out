createvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
movedir_FAILURE CANNOT_MOVE_PARENT_TO_CHILD_DIR
listdir_SUCCESS
makedir_FAILURE INVALID_CHARACTER_IN_DIRNAME
makedir_SUCCESS
makedir_FAILURE DIRECTORY_ALREADY_EXISTS
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY
listdir_SUCCESS
listdir_FAILURE INVALID_FLAG
unmountvfs_SUCCESS
mountvfs_SUCCESS
makedir_FAILURE DIRECTORY_ALREADY_EXISTS
movedir_SUCCESS
movedir_FAILURE VFS_INSUFFICIENT_ARGUMENTS
unmountvfs_SUCCESS
mountvfs_SUCCESS
unmountvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY
listdir_SUCCESS
unmountvfs_SUCCESS
